pom.xml

```diff
@@ -1541,9 +1542,7 @@
                     </execution>
                 </executions>
             </plugin>
-        </plugins>
-        <pluginManagement>
-            <plugins>
+
```

Comment:
@dpursehouse after generating the eclipse files using eclipse:eclipse are you still importing the project using 'existing maven projects' in the Import dialog? You will need to instead use 'Existing Projects into Workspace' under the 'general' folder in that dialog.